Cargo.toml

```diff
@@ -51,3 +51,5 @@ windows-sys = {version = "0.36.1", features = ["Win32_Foundation", "Win32_System
 env_logger = "0.8"
 rand = "0.8.5"
 tiny_http = "0.8.0"
+
+# some-diff
```

Comment:
Is it required?

Reply:
very valid point, this is a remainder from some testing, 
and I should have caught it in my typical self-review... sorry about that.

Reply:
Happens to everyone. :D